### PR TITLE
Include `%(DocumentName)` when invoking the default document generator tool

### PR DESCRIPTION
**Pull request by dougbu**
- #8319

### CODE REVIEW

**Comment by dougbu** [comment]
Getting this one-liner in before we move this part of the feature to the server / web app project.

**Comment by benaadams** [comment]
CI failure https://github.com/aspnet/AspNetCore/issues/8987

**Comment by dougbu** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=140196

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact ryanbrandenburg.

I've triaged the above build.

File: src/Mvc/Extensions.ApiDescription.Design/src/build/Microsoft.Extensions.ApiDescription.Design.targets [hunk 1]
```diff
@@ -187,7 +187,7 @@
             Condition="'%(GenerateDefaultDocumentOptions)' == ''">$(GenerateDefaultDocumentDefaultOptions)</GenerateDefaultDocumentOptions>
       </CurrentServiceProjectReference>
       <CurrentServiceProjectReference Update="@(CurrentServiceProjectReference)">
-        <Command>%(Command) --framework %(TargetFramework) --output %(DocumentPath)</Command>
+        <Command>%(Command) --documentName %(DocumentName) --framework %(TargetFramework) --output %(DocumentPath)</Command>
```
Comment (pranavkm) [inline src/Mvc/Extensions.ApiDescription.Design/src/build/Microsoft.Extensions.ApiDescription.Design.targets]: Assuming these are command line parameters, do these need to be quoted?

Comment (dougbu) [reply src/Mvc/Extensions.ApiDescription.Design/src/build/Microsoft.Extensions.ApiDescription.Design.targets]: Probably wouldn't hurt 😺 for `%(DocumentName)`, `%(DocumentPath)` and `%(ProjectExtensionsPath)` (below)

**Review by pranavkm — APPROVED** [review]

**Pull request merged**